test/TPM_Didymos/TPM_Didymos.jl

```diff
@@ -89,7 +89,7 @@
     n_face_shape1 = length(shape1.faces)  # Number of faces of Didymos
     n_face_shape2 = length(shape2.faces)  # Number of faces of Dimorphos
     
-    ##= Thermal properties =##
+    ##= Thermal properties of Didymos & Dimorphos [c.f. Michel+2016; Naidu+2020] =##
```

Comment:
```suggestion
    ##= Thermal properties of Didymos & Dimorphos [cf. Michel+2016; Naidu+2020] =##
```
https://en.wikipedia.org/wiki/Cf.